low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -243,16 +248,16 @@ class SSPLDevDeploy:
 
     def setup_kafka(self):
         """Setup kafka"""
-        file_path = (f"{self.CORTX_MONITOR_BASE_URL}"
-                     "/low-level/files/opt/seagate/sspl/setup/standalone")
-        kafka_version = '2.13-2.7.0'
-        r = urllib.request.urlopen(f"{self.build_url}/3rd_party/commons/"
-            f"kafka/kafka_{kafka_version}.tgz")
-        with open(f"/tmp/kafka_{kafka_version}.tgz", "wb") as f:
-            shutil.copyfileobj(r, f)
-        tar = tarfile.open(f"/tmp/kafka_{kafka_version}.tgz")
-        tar.extractall("/opt/kafka")
-        tar.close()
+        build_url = f"{self.build_url}/3rd_party/commons/kafka/"
```

Comment:
This is fine. 
In case build_url is not passed in command line, we should still install it from yum.
So add a check here.
if build_url:
    yum install -y {build_url}{pkg}
else:
    yum install -y kafka

Reply:
ohh.. but I see it hardcoded.. 
`self.build_url = ("http://cortx-storage.colo.seagate.com/releases"
                          "/cortx/github/main/centos-7.8.2003"
                          "/last_successful_prod/")`

Reply:
Remove installed kafka and just try with local RPMS installation through sspl_dev_deploy.
Ensure this case is covered.

Reply:
yes it works fine.